src/react-vite/03-intro-to-jsx/intro-to-jsx.md

```diff
@@ -2,33 +2,46 @@
 @parent learn-react-vite 3
 @outline 3
 
-@description TODO
+@description Learn how to use JSX to define your UI in React.
 
 @body
 
 ## Overview
 
-TODO
+In this section we will:
 
-## Objective 1
+- Learn to use JSX to define our UI
+- Install project assets
+- Create our first component
+- Add variables to JSX
+- Work with loops in JSX
 
-TODO
+## Objective 1 - Creating a UI with JSX
 
 ### Key concepts
 
-TODO
+- What is JSX
+- Loops in JSX
 
-#### Concept 1
+## What is JSX?
 
-TODO
+JSX is used by React developers to define the user interface.
 
-#### Concept 2
+JSX is a special syntax designed to look almost identical to HTML. Developers define the UI using JSX and React transforms it into the HTML the browser displays. JSX can be written alongside standard JavaScript, which makes for a powerful programming environment.
 
-TODO
+React has a procedural `React.createElement` syntax, but most applications do not use it directly. Instead, views are defined and maintained in JSX and will automatically be transpiled into the equivalent `React.createElement` calls at build-time.
 
-### Requirements
+## Exercise #1
+
+### Acceptance criteria
+
+- Create the homepage using JSX
+- Match the design below
 
-TODO
+<img src="../static/img/react-vite/mock-homepage.jpg"
+  style="border: solid 1px black; max-width: 640px;"/>
+
+### Requirements
```

Comment:
Given our convo this morning, I think these sections can now be combined into an “Exercise” section right above “Solution”.